docs/source/technical_tutorials/visualization/Visualizing OPERA-DISP tile with TiTiler-MultiDim.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p>I would just do something like `assert nc_key, "No netcdf downloaded to {download_dir}!" before defining nc_key.</p><p> </p><p>Then you confidently assume nc_path exists and you won't need the if/else control flow for nc_key</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/520/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>